### ISSUE TITLE
Fix crash on Linux 

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/ps#47]
https://travis-ci.org/r-lib/ps/jobs/438279198#L2369

**Comment by gaborcsardi** [comment]
```
root@01394eeee9b0:~/home/works/ps# R -q -e 'devtools::test(filter = "common")'
> devtools::test(filter = "common")
Loading ps
Loading required package: testthat
Testing ps
✔ | OK F W S | Context
⠼ | 65       | common*** Error in `/usr/local/lib/R/bin/exec/R': free(): invalid pointer: 0x000055cb6a9073c0 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x70bfb)[0x7f74a09d4bfb]
/lib/x86_64-linux-gnu/libc.so.6(+0x76fc6)[0x7f74a09dafc6]
/lib/x86_64-linux-gnu/libc.so.6(+0x7780e)[0x7f74a09db80e]
/root/home/works/ps/src/ps.so(ps__read_file+0xcb)[0x7f74900eb74b]
/root/home/works/ps/src/ps.so(psll__parse_stat_file+0x69)[0x7f74900ebe59]
/root/home/works/ps/src/ps.so(psll_linux_ctime+0x2f)[0x7f74900ec26f]
/root/home/works/ps/src/ps.so(psll_is_running+0x2d)[0x7f74900ec95d]
/usr/local/lib/R/lib/libR.so(+0xf2c6b)[0x7f74a123fc6b]
/usr/local/lib/R/lib/libR.so(Rf_eval+0x78d)[0x7f74a12838ed]
/usr/local/lib/R/lib/libR.so(+0x13963e)[0x7f74a128663e]
/usr/local/lib/R/lib/libR.so(Rf_eval+0x564)[0x7f74a12836c4]
```

**Comment by gaborcsardi** [comment]
Fixed by ddc87ca9eb8a03a5756fa8ebb63d85c5e87bf45c
